client/metrics.go

```diff
@@ -23,6 +23,11 @@ import (
 // make sure register metrics only once
 var initialized int32
 
+func init() {
+	initMetrics(prometheus.Labels{})
+	initCmdDurations()
```

Comment:
Do we still need to call it inside `initAndRegisterMetrics`?

Reply:
yesh, if callers want to add const labels, we should reinitialize variables.